api/sheets/get_name.md

```diff
@@ -22,14 +22,19 @@ Given a cell or a column range of cells with [DCIDs](/glossary.md) (unique ident
 
 Before trying this method out, make sure to follow the setup directions in [the main section for Sheets docs](/api/sheets/index.html).
 
-### Example 1: Retrieve the name of a country by its DCID.
+### Example 1: Retrieve the name of a country by its DCID
```

Comment:
keep the periods here and below? it's been used this way throughout our docs as we rewrote them with annie.

Reply:
Done